### PR TITLE
Upgrade axios version

### PR DESCRIPTION
**Pull request by sugendran**
Upgrade Axios to the latest version.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1140,13 +1140,22 @@ axe-core@^4.0.2:
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-4.4.1.tgz#7dbdc25989298f9ad006645cd396782443757413"
   integrity sha512-gd1kmb21kwNuWr6BQz8fv6GNECPBnUasepcoLbekws23NVBLODdsClRZ+bQ8+9Uomf3Sm3+Vwn0oYG9NvwnJCw==
 
-axios@>=0.13.0, axios@^0.26.1:
+axios@>=0.13.0:
```
Comment (sugendran) [inline yarn.lock:1143]: Can't work out why it's got this in the lock file - aint nottin requiring it

Comment (sugendran) [reply yarn.lock:1143]: Ah... it's a dev dependency

├─ @types/moxios@0.4.15
│  └─ axios@>=0.13.0

Comment (sugendran) [reply yarn.lock:1143]: Last published 7 years ago :(

Comment (sugendran) [reply yarn.lock:1143]: Looks like it's abandoned. https://github.com/axios/moxios/pull/91

**Review by johndagostino — APPROVED** [review]

**Pull request merged**